src/components/LoaderError/LoaderError.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const LoadingError: React.FC = () => (
```

Comment:
The component name should match the file name. Rename the component to 'LoaderError' or change the file name to 'LoadingError.tsx'